templates/zerver/api/update-display-settings.md

```diff
@@ -0,0 +1,35 @@
+# Update display settings
+
+{generate_api_description(/settings/display:patch)}
+
+## Usage examples
+
+{start_tabs}
+{tab|python}
+
+{generate_code_example(python)|/settings/display:patch|example}
+
+{tab|curl}
+
+{generate_code_example(curl, include=["left_side_userlist", "emojiset"])|/settings/display:patch|example}
+
+{end_tabs}
+
+## Parameters
+
+{generate_api_arguments_table|zulip.yaml|/settings/display:patch}
+
+## Response
+
+#### Return values
+
+The server will return the settings that have been changed after the request,
+with their new value. Please note that this doesn't necessarily mean that it
+will return all the settings passed as parameters in the request, but only
+those ones that were different than the already existing setting.
```

Comment:
Should this be "... different *from*"? Reference: https://english.stackexchange.com/a/845. In the codebase too, there are ~28 of "different from" and just [one](https://github.com/zulip/zulip/blob/master/templates/zerver/api/update-notification-settings.md) of "different than".

Reply:
yes, agreed.

Reply:
Made these changes.